earthnet_models_pytorch/model/channel_u_net.py

```diff
@@ -1,140 +0,0 @@
-"""Channel-U-Net
```

Comment:
This file was probably unintentionally deleted, or is there another reason?

Reply:
Yes, it is useless, I tried some things, but it was not great.